src/ports/postgres/modules/recursive_partitioning/random_forest.sql_in

```diff
@@ -208,13 +208,26 @@ forest_train(training_table_name,
 
     <tr>
     <th>dependent_var_levels</th>
-    <td>itext. For classification, the distinct levels of the dependent variable.</td>
+    <td>text. For classification, the distinct levels of the dependent variable.</td>
     </tr>
 
     <tr>
     <th>dependent_var_type</th>
     <td>text. The type of dependent variable.</td>
     </tr>
+
+    <tr>
+    <th>independent_var_types</th>
+    <td>text. A comma separated string for the types of independent variables.</td>
+    </tr>
+
+    <tr>
+    <th>null_proxy</th>
+    <td>text. Describes how NULLs are handled.  If NULL is not 
+    treated as a separate categorical variable, this will be blank.
```

Comment:
again `this will be NULL` is more appropriate.

Reply:
thanks I made the suggested changes